pom.xml

```diff
@@ -133,6 +130,29 @@
       <version>5.4.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-netty-shaded</artifactId>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-all</artifactId>
+      <version>1.49.0</version>
+      <exclusions>
+        <exclusion>
+          <artifactId>io.grpc</artifactId>
+          <groupId>grpc-netty</groupId>
+        </exclusion>
+      </exclusions>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.perfmark</groupId>
+      <artifactId>perfmark-api</artifactId>
```

Comment:
Is `perfmark-api` a requirement for datastream client?

Reply:
yes this is a requirement for datastream client library